backend/routers/services.py

```
@@ -2,67 +2,77 @@
 from db import get_db
 from typing import List, Dict, Any
 import json
+import logging
+
+# Import dynamic comprehensive pricing
+try:
+    from dynamic_comprehensive_pricing import DynamicComprehensivePricing
```

Ruff (0.12.2)

`dynamic_comprehensive_pricing.DynamicComprehensivePricing` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)